dialogflow.cx.CreateIntentIT: testCreateIntentRegional failed

Note: #186 was also for this test, but it was closed more than 10 days ago. So, I didn't mark it flaky.

----

commit: 8a5412417900edbfafa967b2370582fb72fc67e1
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/7ffffbb6-43fe-41ce-9a00-80985b556c9d), [Sponge](http://sponge2/7ffffbb6-43fe-41ce-9a00-80985b556c9d)
status: failed
<details><summary>Test output</summary><br><pre>java.lang.NullPointerException
	at dialogflow.cx.CreateIntentIT.testCreateIntentRegional(CreateIntentIT.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:364)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:272)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:237)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:158)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
</pre></details>

```
[INFO] Running dialogflow.cx.PageManagementIT
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.04 s - in dialogflow.cx.PageManagementIT
[INFO] Running dialogflow.cx.CreateAgentIT
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.352 s - in dialogflow.cx.CreateAgentIT
[INFO] Running dialogflow.cx.ExportAgentIT
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 7.499 s - in dialogflow.cx.ExportAgentIT
[INFO] Running dialogflow.cx.DetectIntentIT
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.6 s - in dialogflow.cx.DetectIntentIT
[INFO] Running dialogflow.cx.DetectIntentStreamIT
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.588 s - in dialogflow.cx.DetectIntentStreamIT
[INFO] Running dialogflow.cx.CreateFlowIT
Mar 29, 2022 10:52:49 AM io.grpc.internal.ManagedChannelOrphanWrapper$ManagedChannelReference cleanQueue
SEVERE: *~*~*~ Channel ManagedChannelImpl{logId=9, target=dialogflow.googleapis.com:443} was not shutdown properly!!! ~*~*~*
    Make sure to call shutdown()/shutdownNow() and wait until awaitTermination() returns true.
java.lang.RuntimeException: ManagedChannel allocation site

Mar 29, 2022 10:52:49 AM io.grpc.internal.ManagedChannelOrphanWrapper$ManagedChannelReference cleanQueue
SEVERE: *~*~*~ Channel ManagedChannelImpl{logId=21, target=global-dialogflow.googleapis.com:443} was not shutdown properly!!! ~*~*~*
    Make sure to call shutdown()/shutdownNow() and wait until awaitTermination() returns true.
java.lang.RuntimeException: ManagedChannel allocation site
	at io.grpc.internal.ManagedChannelOrphanWrapper$ManagedChannelReference.<init>(ManagedChannelOrphanWrapper.java:93)


[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.621 s - in dialogflow.cx.CreateFlowIT
[INFO] Running dialogflow.cx.CreateIntentIT
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.187 s <<< FAILURE! - in dialogflow.cx.CreateIntentIT
[ERROR] dialogflow.cx.CreateIntentIT.testCreateIntentRegional  Time elapsed: 0.473 s  <<< ERROR!
java.lang.NullPointerException
	at dialogflow.cx.CreateIntentIT.testCreateIntentRegional(CreateIntentIT.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)

[INFO] Running dialogflow.cx.CreatePageIT
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.139 s - in dialogflow.cx.CreatePageIT
[INFO] Running dialogflow.cx.ListTestCaseResultsIT
Mar 29, 2022 10:52:53 AM io.grpc.internal.ManagedChannelOrphanWrapper$ManagedChannelReference cleanQueue
SEVERE: *~*~*~ Channel ManagedChannelImpl{logId=1, target=global-dialogflow.googleapis.com:443} was not shutdown properly!!! ~*~*~*
    Make sure to call shutdown()/shutdownNow() and wait until awaitTermination() returns true.
java.lang.RuntimeException: ManagedChannel allocation site
	at io.grpc.internal.ManagedChannelOrphanWrapper$ManagedChannelReference.<init>(ManagedChannelOrphanWrapper.java:93)
	at io.grpc.internal.ManagedChannelOrphanWrapper.<init>(ManagedChannelOrphanWrapper.java:53)
	at io.grpc.internal.ManagedChannelOrphanWrapper.<init>(ManagedChannelOrphanWrapper.java:44)
	at io.grpc.internal.ManagedChannelImplBuilder.build(ManagedChannelImplBuilder.java:630)

[INFO] Running dialogflow.cx.CreateFlowIT
Mar 29, 2022 10:53:18 AM io.grpc.internal.ManagedChannelOrphanWrapper$ManagedChannelReference cleanQueue
SEVERE: *~*~*~ Channel ManagedChannelImpl{logId=21, target=global-dialogflow.googleapis.com:443} was not shutdown properly!!! ~*~*~*
    Make sure to call shutdown()/shutdownNow() and wait until awaitTermination() returns true.
java.lang.RuntimeException: ManagedChannel allocation site
	at io.grpc.internal.ManagedChannelOrphanWrapper$ManagedChannelReference.<init>(ManagedChannelOrphanWrapper.java:93)
```

amongst other errors.

--- a/README.md
+++ b/README.md
@@ -22,20 +22,20 @@ If you are using Maven, add this to your pom.xml file:
 <dependency>
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-dialogflow-cx</artifactId>
-  <version>0.13.0</version>
+  <version>0.13.1</version>
 </dependency>
 ```
 
 If you are using Gradle without BOM, add this to your dependencies
 
 ```Groovy
-implementation 'com.google.cloud:google-cloud-dialogflow-cx:0.13.0'
+implementation 'com.google.cloud:google-cloud-dialogflow-cx:0.13.1'
 ```
 
 If you are using SBT, add this to your dependencies
 
 ```Scala
-libraryDependencies += "com.google.cloud" % "google-cloud-dialogflow-cx" % "0.13.0"
+libraryDependencies += "com.google.cloud" % "google-cloud-dialogflow-cx" % "0.13.1"
 ```
 
 ## Authentication

--- a/samples/snippets/src/test/java/dialogflow/cx/CreateAgentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/CreateAgentIT.java
@@ -52,7 +52,7 @@ public class CreateAgentIT {
 
     client.deleteAgent(CreateAgentIT.agentPath);
 
-    // Small delay to prevent reaching quota limit of requests per minute  
+    // Small delay to prevent reaching quota limit of requests per minute
     Thread.sleep(250);
   }
 

--- a/samples/snippets/src/test/java/dialogflow/cx/CreateAgentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/CreateAgentIT.java
@@ -51,6 +51,9 @@ public class CreateAgentIT {
     AgentsClient client = AgentsClient.create(agentsSettings);
 
     client.deleteAgent(CreateAgentIT.agentPath);
+
+    // Small delay to prevent reaching quota limit of requests per minute  
+    Thread.sleep(250);
   }
 
   @Test

--- a/samples/snippets/src/test/java/dialogflow/cx/CreateAgentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/CreateAgentIT.java
@@ -43,7 +43,7 @@ public class CreateAgentIT {
   }
 
   @After
-  public void tearDown() throws IOException {
+  public void tearDown() throws IOException, InterruptedException {
     System.setOut(originalOut);
     String apiEndpoint = "global-dialogflow.googleapis.com:443";
 

--- a/samples/snippets/src/test/java/dialogflow/cx/CreateFlowIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/CreateFlowIT.java
@@ -69,8 +69,8 @@ public class CreateFlowIT {
         flowsClient.deleteFlow(newFlowNameRegional);
       }
 
-    // Small delay to prevent reaching quota limit of requests per minute  
-    Thread.sleep(250);
+      // Small delay to prevent reaching quota limit of requests per minute
+      Thread.sleep(250);
     }
   }
 

--- a/samples/snippets/src/test/java/dialogflow/cx/CreateFlowIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/CreateFlowIT.java
@@ -68,6 +68,9 @@ public class CreateFlowIT {
       try (FlowsClient flowsClient = FlowsClient.create(flowsSettings)) {
         flowsClient.deleteFlow(newFlowNameRegional);
       }
+
+    // Small delay to prevent reaching quota limit of requests per minute  
+    Thread.sleep(250);
     }
   }
 

--- a/samples/snippets/src/test/java/dialogflow/cx/CreateIntentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/CreateIntentIT.java
@@ -93,7 +93,6 @@ public class CreateIntentIT {
         CreateIntent.createIntent(
             DISPLAY_NAME, PROJECT_ID, LOCATION_REGIONAL, AGENT_ID_REGIONAL, TRAINING_PHRASES_PARTS);
     newIntentNameRegional = result.getName();
-    System.out.println("intent name new:" + newIntentNameRegional);
 
     assertEquals(result.getTrainingPhrasesCount(), TRAINING_PHRASES_PARTS.size());
     for (TrainingPhrase trainingPhrase : result.getTrainingPhrasesList()) {

--- a/samples/snippets/src/test/java/dialogflow/cx/CreateIntentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/CreateIntentIT.java
@@ -71,7 +71,7 @@ public class CreateIntentIT {
       }
     }
 
-    // Small delay to prevent reaching quota limit of requests per minute  
+    // Small delay to prevent reaching quota limit of requests per minute
     Thread.sleep(250);
   }
 

--- a/samples/snippets/src/test/java/dialogflow/cx/CreateIntentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/CreateIntentIT.java
@@ -70,6 +70,9 @@ public class CreateIntentIT {
         intentsClient.deleteIntent(newIntentNameRegional);
       }
     }
+
+    // Small delay to prevent reaching quota limit of requests per minute  
+    Thread.sleep(250);
   }
 
   @Test

--- a/samples/snippets/src/test/java/dialogflow/cx/CreatePageIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/CreatePageIT.java
@@ -57,6 +57,9 @@ public class CreatePageIT {
       try (PagesClient pagesClient = PagesClient.create()) {
         pagesClient.deletePage(newPageNameGlobal);
       }
+
+    // Small delay to prevent reaching quota limit of requests per minute  
+    Thread.sleep(250);
     }
 
     // Delete the newly created Page in the regional location.

--- a/samples/snippets/src/test/java/dialogflow/cx/CreatePageIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/CreatePageIT.java
@@ -58,8 +58,8 @@ public class CreatePageIT {
         pagesClient.deletePage(newPageNameGlobal);
       }
 
-    // Small delay to prevent reaching quota limit of requests per minute  
-    Thread.sleep(250);
+      // Small delay to prevent reaching quota limit of requests per minute
+      Thread.sleep(250);
     }
 
     // Delete the newly created Page in the regional location.

--- a/samples/snippets/src/test/java/dialogflow/cx/DetectIntentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/DetectIntentIT.java
@@ -47,7 +47,7 @@ public class DetectIntentIT {
 
   @After
   public void tearDown() throws InterruptedExpection {
-    // Small delay to prevent reaching quota limit of requests per minute  
+    // Small delay to prevent reaching quota limit of requests per minute
     Thread.sleep(250);
   }
 

--- a/samples/snippets/src/test/java/dialogflow/cx/DetectIntentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/DetectIntentIT.java
@@ -45,6 +45,12 @@ public class DetectIntentIT {
   private static String LANGUAGE_CODE = "en-US";
   private static List<String> TEXTS = Arrays.asList("hello", "book a meeting room");
 
+  @After
+  public void tearDown() throws InterruptedExpection {
+    // Small delay to prevent reaching quota limit of requests per minute  
+    Thread.sleep(250);
+  }
+
   @Test
   public void testDetectIntentGlobal() throws Exception {
     Map<String, QueryResult> queryResults =

--- a/samples/snippets/src/test/java/dialogflow/cx/DetectIntentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/DetectIntentIT.java
@@ -23,6 +23,7 @@ import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
+import org.junit.After;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
@@ -46,7 +47,7 @@ public class DetectIntentIT {
   private static List<String> TEXTS = Arrays.asList("hello", "book a meeting room");
 
   @After
-  public void tearDown() throws InterruptedExpection {
+  public void tearDown() throws InterruptedException {
     // Small delay to prevent reaching quota limit of requests per minute
     Thread.sleep(250);
   }

--- a/samples/snippets/src/test/java/dialogflow/cx/DetectIntentStreamIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/DetectIntentStreamIT.java
@@ -59,7 +59,7 @@ public class DetectIntentStreamIT {
     System.setOut(original);
     bout.reset();
 
-    // Small delay to prevent reaching quota limit of requests per minute  
+    // Small delay to prevent reaching quota limit of requests per minute
     Thread.sleep(250);
   }
 

--- a/samples/snippets/src/test/java/dialogflow/cx/DetectIntentStreamIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/DetectIntentStreamIT.java
@@ -58,6 +58,9 @@ public class DetectIntentStreamIT {
   public void tearDown() {
     System.setOut(original);
     bout.reset();
+
+    // Small delay to prevent reaching quota limit of requests per minute  
+    Thread.sleep(250);
   }
 
   @Test

--- a/samples/snippets/src/test/java/dialogflow/cx/DetectIntentStreamIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/DetectIntentStreamIT.java
@@ -55,7 +55,7 @@ public class DetectIntentStreamIT {
   }
 
   @After
-  public void tearDown() {
+  public void tearDown() throws InterruptedException {
     System.setOut(original);
     bout.reset();
 

--- a/samples/snippets/src/test/java/dialogflow/cx/ExportAgentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/ExportAgentIT.java
@@ -81,6 +81,9 @@ public class ExportAgentIT {
 
     client.deleteAgent(ExportAgentIT.agentPath);
     client.close();
+
+    // Small delay to prevent reaching quota limit of requests per minute  
+    Thread.sleep(250);
   }
 
   @Test

--- a/samples/snippets/src/test/java/dialogflow/cx/ExportAgentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/ExportAgentIT.java
@@ -71,7 +71,7 @@ public class ExportAgentIT {
   }
 
   @After
-  public void tearDown() throws IOException {
+  public void tearDown() throws IOException, InterruptedException {
     stdOut = null;
     System.setOut(null);
     String apiEndpoint = "global-dialogflow.googleapis.com:443";

--- a/samples/snippets/src/test/java/dialogflow/cx/ExportAgentIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/ExportAgentIT.java
@@ -82,7 +82,7 @@ public class ExportAgentIT {
     client.deleteAgent(ExportAgentIT.agentPath);
     client.close();
 
-    // Small delay to prevent reaching quota limit of requests per minute  
+    // Small delay to prevent reaching quota limit of requests per minute
     Thread.sleep(250);
   }
 

--- a/samples/snippets/src/test/java/dialogflow/cx/ListTestCaseResultsIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/ListTestCaseResultsIT.java
@@ -45,6 +45,9 @@ public class ListTestCaseResultsIT {
   @After
   public void tearDown() throws IOException {
     System.setOut(originalOut);
+
+    // Small delay to prevent reaching quota limit of requests per minute  
+    Thread.sleep(250);
   }
 
   @Test

--- a/samples/snippets/src/test/java/dialogflow/cx/ListTestCaseResultsIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/ListTestCaseResultsIT.java
@@ -43,7 +43,7 @@ public class ListTestCaseResultsIT {
   }
 
   @After
-  public void tearDown() throws IOException {
+  public void tearDown() throws IOException, InterruptedException {
     System.setOut(originalOut);
 
     // Small delay to prevent reaching quota limit of requests per minute

--- a/samples/snippets/src/test/java/dialogflow/cx/ListTestCaseResultsIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/ListTestCaseResultsIT.java
@@ -46,7 +46,7 @@ public class ListTestCaseResultsIT {
   public void tearDown() throws IOException {
     System.setOut(originalOut);
 
-    // Small delay to prevent reaching quota limit of requests per minute  
+    // Small delay to prevent reaching quota limit of requests per minute
     Thread.sleep(250);
   }
 

--- a/samples/snippets/src/test/java/dialogflow/cx/ListTrainingPhrasesTest.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/ListTrainingPhrasesTest.java
@@ -43,6 +43,9 @@ public class ListTrainingPhrasesTest {
   public void tearDown() throws IOException {
     stdOut = null;
     System.setOut(null);
+
+    // Small delay to prevent reaching quota limit of requests per minute  
+    Thread.sleep(250);
   }
 
   @Test

--- a/samples/snippets/src/test/java/dialogflow/cx/ListTrainingPhrasesTest.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/ListTrainingPhrasesTest.java
@@ -40,7 +40,7 @@ public class ListTrainingPhrasesTest {
   }
 
   @After
-  public void tearDown() throws IOException {
+  public void tearDown() throws IOException, InterruptedException {
     stdOut = null;
     System.setOut(null);
 

--- a/samples/snippets/src/test/java/dialogflow/cx/ListTrainingPhrasesTest.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/ListTrainingPhrasesTest.java
@@ -44,7 +44,7 @@ public class ListTrainingPhrasesTest {
     stdOut = null;
     System.setOut(null);
 
-    // Small delay to prevent reaching quota limit of requests per minute  
+    // Small delay to prevent reaching quota limit of requests per minute
     Thread.sleep(250);
   }
 

--- a/samples/snippets/src/test/java/dialogflow/cx/PageManagementIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/PageManagementIT.java
@@ -73,6 +73,9 @@ public class PageManagementIT {
     AgentsClient client = AgentsClient.create(agentsSettings);
 
     client.deleteAgent(parent);
+
+    // Small delay to prevent reaching quota limit of requests per minute  
+    Thread.sleep(250);
   }
 
   @Test

--- a/samples/snippets/src/test/java/dialogflow/cx/PageManagementIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/PageManagementIT.java
@@ -65,7 +65,7 @@ public class PageManagementIT {
   }
 
   @AfterClass
-  public static void tearDown() throws IOException {
+  public static void tearDown() throws IOException, InterruptedException {
     String apiEndpoint = "global-dialogflow.googleapis.com:443";
     String parentPath = "projects/" + PROJECT_ID + "/locations/global";
 

--- a/samples/snippets/src/test/java/dialogflow/cx/PageManagementIT.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/PageManagementIT.java
@@ -74,7 +74,7 @@ public class PageManagementIT {
 
     client.deleteAgent(parent);
 
-    // Small delay to prevent reaching quota limit of requests per minute  
+    // Small delay to prevent reaching quota limit of requests per minute
     Thread.sleep(250);
   }
 

--- a/samples/snippets/src/test/java/dialogflow/cx/UpdateIntentTest.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/UpdateIntentTest.java
@@ -74,7 +74,7 @@ public class UpdateIntentTest {
   }
 
   @After
-  public void tearDown() throws IOException {
+  public void tearDown() throws IOException, InterruptedException {
     stdOut = null;
     System.setOut(null);
     String apiEndpoint = "global-dialogflow.googleapis.com:443";

--- a/samples/snippets/src/test/java/dialogflow/cx/UpdateIntentTest.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/UpdateIntentTest.java
@@ -85,7 +85,7 @@ public class UpdateIntentTest {
 
     client.deleteAgent(parent);
 
-    // Small delay to prevent reaching quota limit of requests per minute  
+    // Small delay to prevent reaching quota limit of requests per minute
     Thread.sleep(250);
   }
 

--- a/samples/snippets/src/test/java/dialogflow/cx/UpdateIntentTest.java
+++ b/samples/snippets/src/test/java/dialogflow/cx/UpdateIntentTest.java
@@ -84,6 +84,9 @@ public class UpdateIntentTest {
     AgentsClient client = AgentsClient.create(agentsSettings);
 
     client.deleteAgent(parent);
+
+    // Small delay to prevent reaching quota limit of requests per minute  
+    Thread.sleep(250);
   }
 
   @Test

